src/main/resources/application.properties

```diff
@@ -0,0 +1,75 @@
+#Postgres Properties
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL10Dialect
+spring.jpa.properties.hibernate.hbm2ddl.auto=create-drop
+spring.jpa.properties.hibernate.hbm2ddl.import_files=db/init.sql
+spring.datasource.url=jdbc:postgresql://localhost:5432/helpdesk
```

Comment:
usually the host and port are moved as ${} since it can be run in not the same server that application does